pkiinit/Dockerfile

```diff
@@ -23,9 +23,12 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
 RUN set -eux; \
     apk add --no-cache ca-certificates dumb-init
 
-# setup / install the executable (binary or shell)
-COPY pki-init.sh pki-init.sh
-RUN chmod +x pki-init.sh
+RUN mkdir -p /edgex/pkiinit
+
+WORKDIR /edgex
+
+# setup / install the executable
+COPY pki-init pkiinit/pki-init
```

Comment:
I still don't understand why you can't do

`COPY pki-init /edgex/pkiinit/pki-ini` 

and get rid of WORKDIR altogether.

Reply:
workdir specifies the context for the docker,so that later on one does not need to specify the full path of whole directory. Very similar concept to include package in C++, so that the one is not burdened by chaining every places when that package name is changed.

Reply:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/ 
Look at WORKDIR section (excepted): ```For clarity and reliability, you should always use absolute paths for your WORKDIR. Also, you should use WORKDIR instead of proliferating instructions like RUN cd … && do-something, which are hard to read, troubleshoot, and maintain.```

Reply:
Then why not set `WORKDIR /edgex/pkinit` ?  Why `/edgex` ?

Reply:
ok , i can live up with ```/edgex/pkinit``` for now. Nevertheless, we will change that to `/edgex` later on when we have `/edgex/res` directory anyway (for those resources).

Reply:
Ok.  If that's the reason you don't need to change it.